tests/inject-live-reload-test.js

```diff
@@ -95,7 +95,16 @@ describe('dynamicScript returns right script when', hooks => {
     assert.strictEqual(actual, 'http://localhost:35729/_lr/livereload.js?port=35729&host=localhost&path=_lr/livereload');
   });
 
-  it('liveReloadPrefix is provided', assert => {
+  it('liveReloadBaseUrl is provided', assert => {
```

Comment:
Can you make this one `liveReloadBaseURL including port is provided GH#68`?

Reply:
Done! Serves me right for trying to multi-task...